shared/folders/render.desktop.js

```diff
@@ -0,0 +1,131 @@
+// @flow
+import React, {Component} from 'react'
+import type {Props} from './render'
+import {Box, TabBar} from '../common-adapters'
+import {TabBarItem, TabBarButton} from '../common-adapters/tab-bar'
+import List from './list'
+import {globalStyles, globalColors} from '../styles/style-guide'
+
+type State = {
+  showPrivate: boolean
+}
+
+class Render extends Component<void, Props, State> {
+  state: State;
+
+  constructor (props: Props) {
+    super(props)
+
+    this.state = {
+      showPrivate: true
+    }
+  }
+
+  _makeItem (isPublic: boolean) {
+    return <TabBarButton
+      source={{type: 'icon', icon: `subnav-folders-${isPublic ? 'public' : 'private'}`}}
+      style={{
+        ...styleItem,
+        borderBottom: isPublic !== this.state.showPrivate
```

Comment:
Can you just pass an extra argument called `isSelected`, that's what this is doing right? 

Not really sure kinda confusing